sentences/en/light_HassLightSet.yaml

```
@@ -3,73 +3,41 @@ intents:
-          - "[<numeric_value_set>] [(<all>|<the>)] <light> [<in>] <area> to <brightness> [brightness]"
-        response: "brightness"
-
-      - sentences:
-          - "<numeric_value_set> <area> to <brightness>"
-        response: "brightness"
-
-      - sentences:
-          - "[<numeric_value_set>] [the] brightness to <brightness>"
-          - "[<numeric_value_set>] [the] brightness (<in_here>;to <brightness>)"
+          - "[<numeric_value_set>] ([of] [(<all>|<the>)] <location>;[to] <brightness_value>)"
+          - "[<numeric_value_set>] ([of] [(<all>|<the>)] <location>;[the] <brightness_synonyms>) [to] <brightness_value>"
+          - "[<numeric_value_set>] [(<all>|<the>)] <location> ([to] <brightness_value>;[the] <brightness_synonyms>)"
         expansion_rules:
-          in_here: "[in] here"
+          location: "(<in_area_floor>|(<light>;<in_area_floor>))"
         response: "brightness"
-        requires_context:
-          area:
-            slot: true
 
-      # Max/Min brightness
+      # inferred current area control
       - sentences:
-          - "[<numeric_value_set>] <name> brightness to [the] {brightness_level:brightness}"
-          - "[<numeric_value_set>] [the] brightness of <name> to [the] {brightness_level:brightness}"
-          - "[<numeric_value_set>] <name> [to] [the] {brightness_level:brightness} brightness"
-        requires_context:
-          domain: light
-        response: "brightness"
+          - "[<numeric_value_set>] [<the>] <light> [to] <brightness_value>"
+          - "[<numeric_value_set>] [<the>] <light> ([to] <brightness_value>;<here>)"
+          - "[<numeric_value_set>] [(<all>|<the>)] <light> ([to] <brightness_value>;<here>)"
 
-      - sentences:
-          - "[<numeric_value_set>] [the] brightness in <area> to [the] {brightness_level:brightness}"
-          - "[<numeric_value_set>] [the] brightness of <area> to [the] {brightness_level:brightness}"
-          - "[<numeric_value_set>] <area> brightness to [the] {brightness_level:brightness}"
-          - "[<numeric_value_set>] <area> [to] [the] {brightness_level:brightness} brightness"
-        response: "brightness"
+          - "[<numeric_value_set>] [<the>] <light> ([to] <brightness_value>;<brightness_synonyms>)"
+          - "[<numeric_value_set>] [<the>] <light> (([to] <brightness_value>;<brightness_synonyms>);<here>)"
+          - "[<numeric_value_set>] [(<all>|<the>)] <light> (([to] <brightness_value>;<brightness_synonyms>);<here>)"
 
-      - sentences:
-          - "[<numeric_value_set>] [the] brightness to [the] {brightness_level:brightness}"
-          - "[<numeric_value_set>] [the] brightness (<in_here>;to [the] {brightness_level:brightness})"
-        expansion_rules:
-          in_here: "[in] here"
+          - "[<numeric_value_set>] ([to] <brightness_value>;[the] <brightness_synonyms>)"
+          - "[<numeric_value_set>] ([to] <brightness_value>;[the] <brightness_synonyms>;<here>)"
```

This is too permissive, I think
`set to 50% here the brightness`
`set here to 50% the brightness`

Maybe this is better?
```suggestion
          - "[<numeric_value_set>] ([to] <brightness_value>;[the] <brightness_synonyms> [<here>])"
```

---

Makes sense to me. Went ahead and made this suggested change as well and updated a few of the tests accordingly.